src/app.js

```diff
@@ -1 +1,27 @@
 'use strict';
+
+const fs = require('fs');
+
+function copyFiles() {
+  const [fileSource, fileDestination] = process.argv.slice(2);
+
+  if (!fileDestination) {
+    // eslint-disable-next-line no-console
+    console.error('Provide both file source path and file destination path');
+
+    return;
+  }
+
+  if (fileSource === fileDestination) {
```

Comment:
Consider adding a console message or error handling to inform the user that the source and destination paths are the same, which is why the operation is not proceeding.